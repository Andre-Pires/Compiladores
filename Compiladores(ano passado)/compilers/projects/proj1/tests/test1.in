else
if
int
char
return
sizeof
struct
typedef
void
while
->
&&
||
==
!=
>=
<=
;
{
}
=
(
)
&
~
-
+
*
/
<
>
]
[
