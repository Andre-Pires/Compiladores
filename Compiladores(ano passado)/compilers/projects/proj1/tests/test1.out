1 276 else
2 275 if
3 271 int
4 272 char
5 278 return
6 259 sizeof
7 274 struct
8 270 typedef
9 273 void
10 277 while
11 260 ->
12 268 &&
13 269 ||
14 267 ==
15 266 !=
16 264 >=
17 265 <=
18 59 ;
19 123 {
20 125 }
21 61 =
22 40 (
23 41 )
24 38 &
25 126 ~
26 45 -
27 43 +
28 42 *
29 47 /
30 263 <
31 262 >
32 93 ]
33 91 [
#Total Number of Tokens Matched was 33
